.circleci/config.yml

```diff
@@ -3,23 +3,42 @@ version: 2.1
 workflows:
   test:
     jobs:
-      - build-run-linux
+    - build-run-linux:
+        context: hello-world-demos
+
+  test-daily:
+    triggers:
+    - schedule:
+        cron: "0 6 * * *"
```

Comment:
Added this job to run the CI daily so we'll know if the demo ever breaks due to some external change.